bin/minifollowups/pycbc_injection_minifollowup

```diff
@@ -188,23 +236,37 @@ for trig in single_triggers:
                 f'{ifo}/snr',
                 return_indices=True)
 
+# figure out how many injections to follow up
+num_events = int(workflow.cp.get_opt_tags(
+    'workflow-injection_minifollowups',
+    'num-events',
+    ''
+))
 if len(missed) < num_events:
     num_events = len(missed)
 
-# loop over loudest events to be followed up
+# sort the injections
+sort_by = workflow.cp.get_opt_tags(
+    'workflow-injection_minifollowups',
+    'sort-by',
+    ''
+)
+sorted_missed_indexes = sort_injections(args, inj_def, missed, sort_by)
```

Comment:
'indices' ? or just eg 'sorted_missed_idx'

Reply:
In fact `missed` is fine, and keeps the namespace cleaner.